### PR TITLE
Landing Page Design Code Improved

### PR DESCRIPTION
**Pull request by gingerzoealex**
The code for the UI has been broken into components using ReactJS.

### CODE REVIEW

File: services/client/src/App.js [hunk 1]
```diff
@@ -1,84 +1,35 @@
...
-    return (
-      <div>
-        {this.renderHeader()}
-        {this.renderTitles()}
-        <div id="container">
-          {this.renderFirstTime()}
-          {this.renderReturningGuest()}
+        <Router>
+          <Switch>
+            <Route exact path="./pages/Registration/index.js" component={Registration} />
```
Comment (awflwafl) [inline services/client/src/App.js]: Looks like we didn't upload the fix for this yet.

File: services/client/package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
+    "react-navigation": "^2.16.0",
```
Comment (awflwafl) [inline services/client/package.json]: Is this package being used?

File: services/client/src/components/PassOptions/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const Options = () => {
+  return (
+    <div>
+      <div id="options">
+        <div id="returningGuest">
+
+              <span id="clicky">
```
Comment (awflwafl) [inline services/client/src/components/PassOptions/index.js]: Looks like we can still run a code formatter over this file to align indentation.

File: services/client/src/App.js [hunk 4]
```diff
@@ -1,84 +1,35 @@
...
-    return (
-      <div id="returningGuest">
-        <button type="button" onMouseOver="this.displayText('descReturning')"
-          onMouseOut="hideText('descReturning')">Returning Guest</button>
-        <div id="descReturning">
-          <p>Eu iusto oblique deseruisse vel. Falli molestie verterem ei pri, nam sint tamquam ne, quidam senserit ei eum. Illud eloquentiam reformidans nec at. Te vel tollit accusata splendide, has nominati dissentias te, ei altera temporibus intellegebat
-            usu.
-      </p>
-        </div>
+        <Header />
```
Comment (awflwafl) [inline services/client/src/App.js]: Good call splitting this into multiple components.

**Review by awflwafl — COMMENTED** [review]
Looks good overall. I'll be happy to approve after the remaining comments are addressed.

File: services/client/src/App.js [hunk 5]
```diff
@@ -29,7 +29,7 @@ class App extends Component {
         <Router>
           <Switch>
             <Route exact path="/Registration" component={Registration} />
-            <Route exact path="/" component={PageILike} />
+            <Route exact path="/" component={LandingPage} />
```
Comment (gingerzoealex) [inline services/client/src/App.js]: The component name is more relevant @awflwafl

File: services/client/package.json (hunk 2 above)
Comment (gingerzoealex) [reply services/client/package.json]: Removed in latest commit

**Review by gingerzoealex — COMMENTED** [review]
Ok, this commit has the changed files based on @awflwafl's comments.

**Review by awflwafl — APPROVED** [review]

**Pull request merged**